zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -860,6 +860,7 @@ brk.add.title                     = Add breakpoint
 brk.alwaysOnTop.message			  = By default ZAP will remain on top of all other windows when a breakpoint is hit.\n\
 Press 'Cancel' to disable this feature.\n\
 This option can changed via Tools/Breakpoints
+brk.checkBox.fixLength            = Fix content length
```

Comment:
```suggestion
brk.checkBox.fixLength            = Fix Content Length
```

Reply:
Could it be called "update" rather than "fix"?